modules/sdk-coin-ada/src/keyPair.ts

```diff
@@ -11,18 +12,48 @@ export class KeyPair extends Ed25519KeyPair {
     super(source);
   }
 
-  getAddress(format?: AddressFormat): string {
-    throw new Error('Method not implemented.');
+  /**
+   *  @param {string} network
+   *  @returns { CardanoWasm.Address }
+   */
+  getAddress(network): string {
```

Comment:
is network a string? does it match inherited method

Reply:
network is a string, but inherited method doesn't take in a string for network. What should be the preferred way to distinguish between testnet and mainnet?